crates/matrix-crypto-ffi/src/machine.rs

```diff
@@ -31,7 +31,7 @@ use ruma::{
         IncomingResponse,
     },
     events::{
-        key::verification::VerificationMethod, room::encrypted::SyncRoomEncryptedEvent,
+        key::verification::VerificationMethod, room::encrypted::OriginalSyncRoomEncryptedEvent,
```

Comment:
This just means "unredacted" but "original" felt a little better because it's at least not *longer* than "redacted". `FooEvent` is now an enum over `OriginalFooEvent` and `RedactedFooEvent`. This change was introduced in https://github.com/ruma/ruma/pull/1017 as a means to make it much easier to handle redacted events / to make it harder to forget to handle them.